collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -97,6 +100,8 @@
 
     private static boolean EXIT = true;
 
+    private Tracker mTracker;
```

Comment:
I don't think there's any real benefit to having a field for the tracker. It can always be called through `getDefaultTracker()` and won't need to be initialized unless it's the first time that it is accessed.